tests/containers/container_engine.pm

```diff
@@ -111,7 +111,7 @@ sub basic_container_tests {
     assert_script_run("$runtime image rm example.com/tw-commit_test");
 
     ## Test connectivity inside the container
-    assert_script_run("$runtime container exec basic_test_container curl -sfI https://opensuse.org", fail_message => "cannot reach opensuse.org");
+    script_retry("$runtime container exec basic_test_container curl -sfIL https://opensuse.org", retry => 3, delay => 5, fail_message => "cannot reach opensuse.org");
```

Comment:
I would put much bigger delay , let say 60 then we will increase our chances ...

Reply:
It's not timeout, timeout is 90. This is just delay and is not much important. The important part is `retry => 3`.